packages/design-tokens/src/tokens/functional/components/video-player/base.json

```diff
@@ -0,0 +1,46 @@
+{
+  "brand": {
+    "VideoPlayer": {
+      "closedCaption": {
+        "margin": {
+          "value": "0 auto 15px auto"
+        },
+        "text": {
+          "padding": {
+            "value": "10px"
+          }
+        }
+      },
+      "playButton": {
+        "height": {
+          "value": "24px"
+        },
+        "width": {
+          "value": "24px"
+        },
+        "padding": {
+          "value": "25px"
+        }
+      },
+      "range": {
+        "height": {
+          "value": "6px"
+        },
+        "borderRadius": {
+          "value": "3px"
+        },
+        "minWidth": {
+          "value": "75px"
+        }
+      },
+      "transition": {
+        "value": "all 0.4s ease-out"
+      },
```

Comment:
Could we use our global animation timing tokens here instead please? I think I left a related comment somewhere in this PR, so perhaps this is no longer required?

Reply:
This has been updated in this SHA: https://github.com/primer/brand/pull/373/commits/53e6be6f783cb211109e7502de6eb28f3b6699e4